charts/pravega/templates/clusterrolebinding.yaml

```diff
@@ -2,16 +2,16 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
-  name: {{ .Values.serviceAccount.name }}
+  name: {{ template "pravega.fullname" . }}-{{ .Release.Namespace }}
   namespace: {{ .Release.Namespace }}
```

Comment:
Again, `ClusterRoleBinding` is a cluster-wide resource that is not confined to one namespace.

Reply:
fixed